marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/WebSecurityConfiguration.java

```diff
@@ -55,7 +59,13 @@ protected void configure(HttpSecurity http) throws Exception {
             .addFilterAfter(new AuthenticationFilter(hubCentral, hubClientProvider), LogoutFilter.class)
             .csrf().disable()
             // Need to setStatus, sendError causes issues. see https://stackoverflow.com/a/34911131
-            .exceptionHandling().authenticationEntryPoint(((request, response, authException) -> response.setStatus(HttpServletResponse.SC_UNAUTHORIZED)))
+            .exceptionHandling().authenticationEntryPoint(((request, response, authException) -> {
+                if (request.getRequestURI().startsWith("/api/")) {
```

Comment:
noticed a lot of redirect requests for /websocket when running e2e. Filtering /websocket alongwith /api gets rid of the issue. Will probably push the changes after running some more e2e tests

Reply:
This is fixed now.